### ISSUE TITLE
Young tableaux?

### ISSUE THREAD

**Issue by jiahao** [andrioni/Catalan.jl#4]
Can the scope of this package be extended to accommodate some computations to deal with Young tableaux and diagrams? Strictly speaking it seems that these might lie outside the scope of enumerative combinatorics. I've already written some code for my own purposes but it seems more natural to put these into the combinatorics package rather than embed them in RandomMatrices.jl.

**Comment by andrioni** [comment]
Of course! The focus on enumerative combinatorics is mostly because it's what I'm more familiar with, but it should by no means limit the scope of the package. Young tableaux and symmetric functions are two things I'd like to implement here someday.
